homeassistant/components/wemo/fan.py

```diff
@@ -185,31 +160,15 @@ def _update(self, force_update=True):
             self._available = False
             self.wemo.reconnect_with_device()
 
-    #
-    # The fan entity model has changed to use percentages and preset_modes
-    # instead of speeds.
-    #
-    # Please review
-    # https://developers.home-assistant.io/docs/core/entity/fan/
-    #
     def turn_on(
         self,
         speed: str = None,
         percentage: int = None,
```

Comment:
Guess this should also be 
```suggestion
        percentage: Optional[int] = None,
```